src/tracks/tracks.module.ts

```diff
@@ -2,17 +2,20 @@ import { Module } from '@nestjs/common';
 import { MongooseModule } from '@nestjs/mongoose';
 import { BaseService } from '~shared/services';
 
+import { MentorModule } from '../mentor/mentor.module';
 import { UsersModule } from '../users/users.module';
+import { TrackMentor } from './models/track-mentor.entity';
```

Comment:
Codacy found an issue: ['TrackMentor' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-backend/pullRequest?prid=6034169)